### ISSUE TITLE
doubt about the PointNetCls 

### ISSUE THREAD

**Issue by MikeXuPku** [fxia22/pointnet.pytorch#25]
last return sentence in classPointNetCls  is:

return F.log_softmax(x, dim=0), trans

however in my opnion shape(x) should be (batchsize,k),then the last return should be:

return F.log_softmax(x, dim=-1), trans 
or
return F.log_softmax(x, dim=1), trans

I don't know if I just thought wrong, can anyone solve my doubt.

**Comment by fxia22** [comment]
This issue is fixed in the latest master branch, thanks for pointing it out.
